### ISSUE TITLE
1838 novels, #10-14

### ISSUE THREAD

**Issue by riddella** [literary-materials/class-of-1838-novels#2]
random_order | title_id | last_name | title_main | publisher_name | num_chapters | filename | note |  
-- | -- | -- | -- | -- | -- | -- | -- | --
10 | 15891 | Tuite | The Reclaimed Family | Simpkin | 57 | 15891__chp19of57_seed15891.html |   |  
11 | 16715 | Bethune | Tales and Sketches of the Scottish Peasantry | Fraser and Co. | 10 | 16715__chp3of10_seed16715.html | 8 stories; one divided into 3 chapters; ignore poems |  
12 | 15812 | Collingridge | A Night Near Windsor | James Fraser | Intro+7+5 | 15812__collingridge_night_near_windsor_chp13of13_seed15812.html |   |  
13 | 4300 | Smith | Jane Lomax | Henry Colburn | 11+12+12 | 4300__chp18of35_seed4300.html |   |  
14 | 15877 | BTAO "The Jew and his Daughter converted to Christianity" | The Curate's Grave | Richard Moore Tims | 12 | 15877__chp5of12_seed15877.html |   |

**Comment by leschnei** [comment]
Thanks!

**Comment by leschnei** [comment]
Just double checking -- is the randomly selected chapter the "random_order"?

**Comment by leschnei** [comment]
[ReclaimedFamily.txt](https://github.com/literary-materials/class-of-1838-novels/files/2788301/ReclaimedFamily.txt)
Chapter 10

**Comment by riddella** [comment]
Chapter is in the filename. Ignore the "random_order" column.

So, `15891__chp19of57_seed15891` --  so chapter 19

**Comment by leschnei** [comment]
I’m not seeing the file name on the Github page... -- the table above doesn't show it the way that you've sent it to me.

**Comment by riddella** [comment]
Can you scroll the table horizontally, so it shows cells on the far right which are currently off the page?

**Comment by leschnei** [comment]
Yes! Thank you!

**Comment by leschnei** [comment]
[Tuite.txt](https://github.com/literary-materials/class-of-1838-novels/files/2825694/Tuite.txt)

**Comment by leschnei** [comment]
[Bethune.txt](https://github.com/literary-materials/class-of-1838-novels/files/2897574/Bethune.txt)

**Comment by leschnei** [comment]
For "A Night Near Windsor" - am I encoding the chapter from "A Tale of the Turf"? That's the only one of the two collections with 5 chapters, excluding the intro at the very beginning.

**Comment by riddella** [comment]
@leschnei good catch! There are more chapters in that book. I've re-sampled a chapter for the book: encode the last chapter in the entire book (so from A Tale of the Turf).

**Comment by ayarnell** [comment]
@riddella Created PR for the Tuite and Bethune chapters.

**Comment by ayarnell** [comment]
@leschnei There were a couple things that were flagged by the qc script in these texts, and a couple things I noticed weren't following the more recent encoding instructions:
- chapter titles should be commented out
- epigraphs and similar blocks of text are meant to go in `<div>` tags.  You can also mark the end of lines with a `<br>` tag
- empty `<p>` tags will be flagged by the qc script. 
- Comments are written like `<!-- -->` Watch out for spaces between the exclamation point and first dash, that will break the comment. 

Thanks! :-)

**Comment by leschnei** [comment]
Thanks, Amy!

**Comment by riddella** [comment]
@leschnei I'm sorry I forgot to update you about some of these changes. I think when we started things, I likely told you to use, say, empty `<p>` tags. This turned out to be wrong (it violates an HTML5 rule).

I think the guidelines should be accurate now: https://github.com/random-canon/random-canon/blob/master/novel-encoding-instructions.md If you see anything that doesn't make sense or doesn't align with your experience, please let me know.

**Comment by leschnei** [comment]
@riddella Not a problem! Thanks for the clarification!

**Comment by leschnei** [comment]
[Collingridge.txt](https://github.com/literary-materials/class-of-1838-novels/files/2974628/Collingridge.txt)

**Comment by ayarnell** [comment]
@riddella Opened PR #13 for the Collingridge text.

**Comment by leschnei** [comment]
[Smith.txt](https://github.com/literary-materials/class-of-1838-novels/files/3004384/Smith.txt)

**Comment by ayarnell** [comment]
@riddella Opened PR #14 for Smith text.

**Comment by leschnei** [comment]
[CurateGrave.txt](https://github.com/literary-materials/class-of-1838-novels/files/3291402/CurateGrave.txt)
